tests/api/test_purge.cc

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2023- European Centre for Medium-Range Weather Forecasts (ECMWF).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "dasi/api/Dasi.h"
+
+#include "helper.h"
+
+#include "eckit/filesystem/LocalPathName.h"
+
+namespace dasi::testing {
+
+const auto tempPath = eckit::LocalPathName::cwd() + "/tmp.DASI.purge";
+
+//----------------------------------------------------------------------------------------------------------------------
+
+CASE("testing dasi archive") {
+    TempDirectory tempDir(tempPath, false);
+
+    tempDir.write("simple_schema", SIMPLE_SCHEMA);
+
+    const auto cfg = simpleConfig(tempDir, "simple_schema");
+
+    LOG_D("--- [ARCHIVE] ---");
```

Comment:
Where does LOG_D come from?

How does it differ from the standard eckit debug logging macros? Why do we not use those?